tools/db_crashtest.py

```diff
@@ -233,7 +235,10 @@ def get_dbname(test_name):
             print("Running DB cleanup command - %s\n" % cleanup_cmd)
             # Ignore failure
             os.system(cleanup_cmd)
-        os.mkdir(dbname)
+        try:
```

Comment:
Is this for the case where the test dir is created on remote storage and we don't care whether this step succeeds or not?

Reply:
Yes.